homeassistant/components/mailbox/demo.txt

```diff
@@ -0,0 +1,7 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam porta, odio vitae volutpat sollicitudin, neque odio lacinia lorem, non auctor metus lorem id arcu. Integer mollis luctus velit id pharetra. In id sagittis metus. Mauris ultricies, lorem et pulvinar ornare, metus augue iaculis enim, ullamcorper dictum ipsum urna sit amet augue. Nunc dapibus libero quis dui semper scelerisque. Nam dapibus feugiat est ac dapibus. Suspendisse sit amet odio ac turpis commodo volutpat ut ut massa. Etiam quam mauris, tincidunt at orci ut, eleifend aliquam sem.
```

Comment:
Let's not include this file but instead just generate some text on the fly. It's ok to be garbage. Otherwise we're going to ship 2.2kB to every user forever just for a demo.